client/methods/updateMessage.js

```diff
@@ -64,10 +64,12 @@ Meteor.methods({
 					delete messageObject.$set.msg;
 				}
 			}
-			ChatMessage.update({
-				_id: message._id,
-				'u._id': Meteor.userId(),
-			}, { $set : messageObject });
+			if (originalMessage.msg !== message.msg) {
```

Comment:
It looks like you could move this if up to the start of the method and make an "early return", similarly to the other file :)